tasks/common.yml

```diff
@@ -0,0 +1,31 @@
+- name: write client list to temp file for comparison with existing certs
+  copy:
+    content: "{{ clients | join('\n') }}"
+    dest: /tmp/client-list
+  when: openvpn_sync_certs
+
+- name: get existing certs
+  shell: ls {{ openvpn_key_dir }} | grep '\.csr' | grep -v 'server.csr' | sort | cut -f1 -d'.' > /tmp/existing-certs
+  when: openvpn_sync_certs
+
+- name: get difference between 'clients' variable and existing certs
+  shell: |
+      sort -o /tmp/client-list /tmp/client-list
+      diff /tmp/client-list /tmp/existing-certs | grep '>' | awk '{print $2}'
+  register: additional_certs_to_revoke
+  when: openvpn_sync_certs
+
+- name: find clients to add and notify
+  shell: |
+      sort -o /tmp/client-list /tmp/client-list
+      diff /tmp/client-list /tmp/existing-certs | grep '<' | awk '{print $2}'
+  register: additional_certs_to_add
+  when: openvpn_sync_certs
+
+- set_fact:
+    additional_certs_to_revoke: []
+  when: additional_certs_to_revoke is not defined
+
+- set_fact:
+    additional_certs_to_add: []
+  when: additional_certs_to_add is not defined
```

Comment:
This is failing because it doesn't have the `stdout_lines` attribute.

```
TASK [ansible-role-openvpn : generate client key] ******************************
task path: /etc/ansible/roles/ansible-role-openvpn/tasks/client_keys.yml:16
fatal: [127.0.0.1]: FAILED! => {"msg": "'dict object' has no attribute 'stdout_lines'"}
```

Probably fixable with 
```
# spoof the shell output format
additional_certs_to_revoke: 
    stdout_lines: []
```